route/kafkamdm.go

```diff
@@ -96,11 +95,19 @@ func NewKafkaMdm(key string, matcher matcher.Matcher, topic, codec, schemasFile,
 		log.Fatalf("kafkaMdm %q: failed to initialize partitioner. %s", r.key, err)
 	}
 
+	kfkVersion, err := sarama.ParseKafkaVersion(kafkaVersion)
```

Comment:
defaulting to an empty string and showing an error if the user doesn't specify a version, seems messy.
Why don't we default to `sarama.MinVersion.String()` ?